css/area_footnotes.less

```diff
@@ -0,0 +1,65 @@
+/**
+ * This file provides the design styles for the footnotes.
+ *
+ */
+
+/* + + + + + + + + + + + + + + + + + + + + + + + + + + */
+/* footnotes section  */
+.dokuwiki div.footnotes {
+  div.fn {
+    margin-bottom: .2rem;
+    display: table;//
+    vertical-align: top;//
```

Comment:
What's up with these trailing comments?

Reply:
> [09dd0a6](https://github.com/cosmocode/dokuwiki-template-sprintdoc/commit/09dd0a60f884baad73f4a886073f20c16e494520) seems to be completely unrelated and bloats up this PR unnecessarily

reverted (will be part of separate merge request)